src/tools/simeng/gdb/GDBStub.cc

```diff
@@ -0,0 +1,331 @@
+#include "simeng/GDBStub.hh"
+
+namespace simeng {
+
+GDBStub::GDBStub(simeng::Core& core, simeng::MemoryInterface& dataMemory)
+    : core_(core), dataMemory_(dataMemory) {}
+
+int GDBStub::run() {
+  // verbose_ = 1; // uncomment this if you'd like the GDBStub to run verbosely
```

Comment:
Is there a way to handle verbose runs using a runtime or cmake argument rather than having to manually un-comment code?

Reply:
@BraveSirRobbo we could have an additional optional config section for GDB and pass the config file through to the GDBStub to set such options? We wouldn't have to recompile then. I'd imagine, in time, we won't need such functionality as the implementation won't need debugging/fixing.

Reply:
Supporting GDB's run/start commands would be able to support runtime flags, but I haven't managed to implement that (yet?).

I agree with this being a good simple alternative in the meanwhile though - would you like me to take a look at it ? I haven't dealt with the config files yet so I don't know how hard it would be for me

Reply:
If it is an option to pass runtime flags within GDB with the start/run command then I think that would be preferable for this verbose mode. So I think concentrating on getting them to work would be best (if possible)